scripts/debcheck.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+#
+# use dose-debcheck/dose-distcheck to do "repoclosure" for debian
+# Debian package dose-distcheck
+# Fedora package dose3-tools
+
+DEBIAN_RELEASE=${1:-bullseye}
+FOREMAN_VERSION=${2:-nightly}
+# user repos TBD
+
+case ${DEBIAN_RELEASE} in
+  focal)
+    # this is broken, as ubuntu needs multiple archives to be enabled to work
+    DEBIAN_PACKAGES_SOURCE=http://archive.ubuntu.com/ubuntu/dists/${DEBIAN_RELEASE}/main/binary-amd64/Packages.xz
+    ;;
+  *)
+    DEBIAN_PACKAGES_SOURCE=https://deb.debian.org/debian/dists/${DEBIAN_RELEASE}/main/binary-amd64/Packages.xz
+    ;;
+esac
+
+PUPPET_PACKAGES_SOURCE=https://apt.puppet.com/dists/${DEBIAN_RELEASE}/puppet6/binary-amd64/Packages
```

Comment:
Should we default to Puppet 7?

Reply:
technically, we still support 6, so testing it as the lower boundary is more correct?

Reply:
I guess. I also considered a an env var for just the version that allows you to easily test with both. Maybe it's overkill for now.

Reply:
that's actually a good idea, done